Gemfile

```diff
@@ -32,6 +32,7 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem "factory_bot_rails", "~> 6.1"
+  gem "ffaker", "~> 2.18", require: "ffaker"  
```

Comment:
I don't think we need this `require`. That is used when a non-default directory is used by the gem. FFaker docs don't suggest we do this.

Reply:
I actually have questions about this - what's the difference programmatically between placing `require 'ffaker' ` in `/test/test_helper.rb` versus in the individual factories files (e.g. `/test/factories/clients.rb`) versus in Gemfile?

Reply:
The individual factory files seems like the correct location (e.g. /test/factories/clients.rb) , but I'm inferring that because the [docs](https://github.com/ffaker/ffaker) seem to suggest that, not because I actually know what's going on under the hood.

Reply:
The docs are suggesting how you use it outside of rails. Rails does a lot of autoloading for the developer, that's why you don't really see any require statements throughout the code. The test_helper.rb file includes everything all tests need for the entire suite. It's better to have in that file, once, instead of having to require it in every factory.

Reply:
Also, the `require: "ffaker"` here is not the same as `require` used in any ruby file.

Reply:
Thank you!!

I thought the autoloading is accomplished by `Bundler.require(*Rails.groups)` in \config\application.rb ? 

https://www.justinweiss.com/articles/how-does-rails-handle-gems/ 
"`Bundler.require` requires all the gems in all the groups you pass to it."

Anyway I removed the `require` statements that I previously added and re-pushed.